### PR TITLE
Create ui-c8y-1020-2-0-removing-password-change-possibility-for-devices-6314-graft-release-cd-6525.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-59197](https://cumulocity.atlassian.net/browse/MTM-59197)
Original PR: [6525](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6525)

### CODE REVIEW

File: content/change-logs/platform-services/ui-c8y-1020-2-0-removing-password-change-possibility-for-devices-6314-graft-release-cd-6525.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: removing password change possibility for devices (#6314) [GRAFT][release/cd] (#6525)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1020-2-0-removing-password-change-possibility-for-devices-6314-graft-release-cd-6525.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Remove password change option for devices
```

File: content/change-logs/platform-services/ui-c8y-1020-2-0-removing-password-change-possibility-for-devices-6314-graft-release-cd-6525.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: q3kclF6pO
+    label: Authentication
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-59197
+version: 1020.2.0
+---
+removing password change possibility for devices (#6314) [GRAFT][release/cd] (#6525)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1020-2-0-removing-password-change-possibility-for-devices-6314-graft-release-cd-6525.md:17]: The Claude AI suggested to use the following body:
```suggestion
To enhance device security, the option for users to change passwords on devices has been removed. Previously, users were able to change passwords directly on devices, which could potentially lead to unauthorized access or inconsistencies in password management. With this change, device passwords can now only be managed through the central device management system, ensuring better control and security. Existing devices will no longer show the password change option, and users will need to use the device management system for any password-related tasks.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by mmed-softwareag** [comment]
release notes for this is in: https://github.com/SoftwareAG/c8y-docs/pull/2209

**Pull request closed without merging**